src/contexts/Staking/index.tsx

```diff
@@ -301,17 +303,17 @@ export const StakingProvider = ({
       );
 
       if (s === undefined) {
-        statuses[nomination] = 'waiting';
+        statuses[nomination] = t('contexts.waiting');
         continue;
       }
       const exists = (s.others ?? []).find(
         (_o: any) => _o.who === activeAccount
       );
       if (exists === undefined) {
-        statuses[nomination] = 'inactive';
+        statuses[nomination] = t('contexts.inactive');
         continue;
       }
-      statuses[nomination] = 'active';
+      statuses[nomination] = t('contexts.active');
```

Comment:
again, these status codes do not need translating. If anything they should be refactored to an enum,